Analysis script template.R

```diff
@@ -0,0 +1,261 @@
+
+
+# Variable defined to keep track of simulation time (delete afterwards)
+init <- Sys.time()
+
+# Control variables of simulation
+npats <- 500 # Number of patients in simulation
+tx.cost <- 0 # Total treatment cost
```

Comment:
We need to adopt a convention when writing variable names. I do not have a strong preference here but I usually use _ instead of .